inference-engine/src/mkldnn_plugin/emitters/jit_snippets_emitters.hpp

```diff
@@ -75,7 +112,11 @@ class TileEmitter : public jit_emitter {
     TileEmitter(mkldnn::impl::cpu::x64::jit_generator* h, mkldnn::impl::cpu::x64::cpu_isa_t isa,
     const std::shared_ptr<ngraph::Node>& n)
     : jit_emitter(h, isa, n) {
-        code = ngraph::as_type_ptr<ngraph::snippets::op::Tile>(n)->region;
+        const auto kernel = ngraph::as_type_ptr<ngraph::snippets::op::Tile>(n);
+        code = kernel->region;
```

Comment:
`Tile`? And below

Reply:
Took me a while to get your point) Renamed the variable from "kernel" to "tile" and changed the exception message